app/assets/scripts/components/map.js

```diff
@@ -261,7 +261,13 @@ const Map = React.createClass({
 
   renderOverlayLegend: function (scale) {
     const isQuantile = scale.hasOwnProperty('invertExtent');
-    const iterable = (isQuantile ? scale.range() : scale.domain()).sort();
+    let iterable = (isQuantile ? scale.range() : scale.domain()).sort();
+    const category = this.props.overlay.category.toLowerCase();
```

Comment:
If for some reason an indicator didn't have an assigned category, this would throw an error as it would try to lower case an undefined value.

A better way would be:

```(javascript)
let category = get(this.props, 'overlay.category');
if (category) {
  category = category.toLowerCase()
  // do the rest of function
}
```